networkx/convert.py

```diff
@@ -90,13 +90,6 @@ def to_networkx_graph(data, create_using=None, multigraph_input=False):
         except Exception as err:
             raise nx.NetworkXError("Input is not a correct NetworkX graph.") from err
 
-    # pygraphviz  agraph
```

Comment:
This is an unfortunately rendering of the diff... FWIW the check has not been modified at all, it was just moved so that the added edgelist check is first!